ui/src/main/java/edu/wpi/grip/ui/deployment/FRCAdvancedDeploymentOptionsController.java

```diff
@@ -32,11 +34,13 @@ FRCAdvancedDeploymentOptionsController create(
 
     @Inject
     FRCAdvancedDeploymentOptionsController(DeployedInstanceManager.Factory deployedInstanceManagerFactor,
+                                           Pipeline pipeline,
                                            @Assisted Consumer<DeployedInstanceManager> onDeployCallback,
                                            @Assisted("stdOut") Supplier<OutputStream> stdOut,
                                            @Assisted("stdErr") Supplier<OutputStream> stdErr) {
         super("FRC Advanced", onDeployCallback);
         this.deployedInstanceManagerFactor = deployedInstanceManagerFactor;
+        this.projectAddress = pipeline.getProjectSettings().computeNetworkTablesServer();
```

Comment:
Cool, but what if a user wants to override this?  They'd have to change this every time.  We should have `robotAddress` and `networkTablesAddress` as two different project settings